libs/backend/src/ballot_package/ballot_package_io.test.ts

```diff
@@ -35,17 +33,19 @@ import { UsbDrive } from '../get_usb_drives';
 
 const mockFeatureFlagger = getFeatureFlagMock();
 
+jest.mock('@votingworks/auth', (): typeof import('@votingworks/auth') => ({
+  ...jest.requireActual('@votingworks/auth'),
+  authenticateArtifactUsingSignatureFile: jest.fn(),
+}));
+
 jest.mock('@votingworks/utils', (): typeof import('@votingworks/utils') => ({
   ...jest.requireActual('@votingworks/utils'),
   isFeatureFlagEnabled: (flag) => mockFeatureFlagger.isEnabled(flag),
 }));
 
-let mockArtifactAuthenticator: jest.Mocked<ArtifactAuthenticatorApi>;
-
 beforeEach(() => {
+  mockOf(authenticateArtifactUsingSignatureFile).mockResolvedValue(ok());
```

Comment:
As noted in the PR description, I'm still mocking artifact authentication in many places. I have plans to stop mocking in subsequent PRs, where I'll either add signature files to fixtures or just create them on the fly, etc.